.readme-partials.yaml

```diff
@@ -1,7 +1,7 @@
 custom_content: |
   ### Compute alpha to beta migration
   
-  From version 1.6.0-beta `google-cloud-compute` will have the following feature changes: 
+  Java compute library is GA from version 1.7.0 and is completely distinct from the previous 1.x.0 and 0.x.0 versions. From version 1.6.0-beta `google-cloud-compute` will have the following feature changes: 
```

Comment:
"complete distinct" is too strong wording here, also we probably need separate  0.x.x and 1.x.x here.

Essentially it is:
- backwards incompatible with 0.x.x
- is incompatible with 1.5.x-beta and prior in a following way: (and here you keep the description which is already there about LRO and Operation differences).

Reply:
done